changelog/unreleased/add-links-default-permissions-capability.md

```diff
@@ -0,0 +1,6 @@
+Enhancement: Add default permissions capability for links
+
+A capability for default permissions for links has been added.
+
+https://github.com/owncloud/web/issues/9919
+https://github.com/cs3org/reva/pull/4358
```

Comment:
Move the pr above an issue to make the ci happy

Reply:
Got it, thanks!